cli/internal/terraform/terraform/qemu/variables.tf

```diff
@@ -1,3 +1,20 @@
+variable "node_groups" {
+  type = map(object({
+    role           = string
+    instance_count = number // number of instances in the node group
+    disk_size      = number // size of state disk (GiB)
+    vcpus          = number
+    memory         = number // amount of memory per instance (MiB)
+    machine        = string // machine type. use 'q35' for secure boot and 'pc' for non secure boot. See 'qemu-system-x86_64 -machine help'
```

Comment:
```suggestion
```

machine has to be uniform for all node groups and should remain a global option (instead of being chosen per group).